.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,93 @@
+# USTaxes Contributing Guide
+
+We, the maintainers, are really excited that you are interested in contributing to USTaxes. Before submitting your contribution though, please make sure to take a moment and read through the [Code of Conduct](CODE_OF_CONDUCT.md), as well as the appropriate section for the contribution you intend to make:
+
+- [Issue Reporting Guidelines](#issue-reporting-guidelines)
+- [Pull Request Guidelines](#pull-request-guidelines)
+- [Development Guide](#development-guide)
+
+## Maintaining team
+
+* [Aidan Grimshaw](http://github.com/thegrims)
+* [Zak Patterson](http://github.com/zakpatterson)
+
+## Issue Reporting Guidelines
+
+- The issue list of this repo is exclusively for bug reports and feature requests. Please come to the discord with general questions you may have.
+
+- If you have a question, you can get quick answers from the [Discord chat](https://discord.gg/dAaz472mPz).
+
+- If your issue is resolved but still open, please close it. In case you found a solution by yourself, it could be helpful to explain how you fixed it.
+
+- Generally, there are many features still remaining to produce a valid 1040 for most people. We hope you will view this project in light of your individual circumstances and expertise and contribute accordingly! For example if you have a certain type of income or receive a certain credit or deduction, and those are not yet implemented, we would gratefully assist you to implement them!
+
+## Pull Request Guidelines
+
+- Start all commits with a capital letter and imperitive mood, no period at the end.
+
+- Each commit first line should be max 50 characters, followed by a blank line, followed by as much detail as you think is needed, wrapped at 80 characters.
+
+- Github PR title style is the same as commit first line style.
+
+- It's OK to have multiple small commits as you work on the PR - we apply squash merges to all PRs at the end. In particular, you are free to rebase your commits so long as your pull request is in Draft status. As soon as you take it out of draft status, request a review, or receive a review on your pull request, please do not rebase or force push.
```

Comment:
Once the pr is ready for review we or another may pull the branch to test it out, and may want to contribute another assisting commit. So if the pr branch is rebased and force pushed in the meantime, then it will conflict with the other contributor's history. Does that make sense?